src/sentry/event_manager.py

```diff
@@ -1136,6 +1136,30 @@ def _save_aggregate(self, event, hashes, release, **kwargs):
                 event.event_id,
             )
 
+        # Check if already assigned
+        # TODO: convert to model method
+        from sentry.models import GroupAssignee
+        assignee = GroupAssignee.objects.filter(group_id=group.id).first()
+        logger.info('-- starting auto-assignment process --')
+        logger.info('-- project id {}, group id {}'.format(project.id, group.id))
+        logger.info('-- event id {}'.format(event.event_id))
+        logger.info(assignee)
+        if assignee is None or (assignee.user is None and assignee.team is None):
+            from sentry.models import GroupAssignee, ProjectOwnership, Team, User
+            ownership = ProjectOwnership.objects.filter(
+                            project_id=project.id
+                        ).first()
```

Comment:
You could use `ProjectOwnership.objects.get()` here.

Reply:
Since `get` will raise an [exception](https://docs.djangoproject.com/en/2.1/ref/models/querysets/#django.db.models.query.QuerySet.get) if it does not find an object with the given parameters and at this point we are not aware if the project has any ownership rules yet, I thought of using `filter` instead.

Reply:
Changed as suggested.